mungegithub/github/github.go

```diff
@@ -873,12 +873,17 @@ func (config *Config) GetBranchCommits(branch string, limit int) ([]*github.Repo
 }
 
 // GetTokenUsage returns the api token usage of the current github user.
-func (config *Config) GetTokenUsage() (int, error) {
-	limits, _, err := config.client.RateLimits(context.Background())
-	if err != nil {
-		return -1, err
+func (config *Config) GetTokenUsage() int {
+	config.apiLimit.Lock()
+	remaining := config.apiLimit.remaining
+	config.apiLimit.Unlock()
+
+	if config.tokenInUse != "" {
+		// Authenticated token limit.
+		return 5000 - remaining
```

Comment:
Maybe put 5000/60 in constants at the top?

The precise number doesn't matter than much, since we're taking the derivative in Prometheus.

Reply:
Done.